### DISCUSSION TITLE
How to cast between Godot class types?

### DISCUSSION THREAD

**Discussion by ullanar** [quaadgras/graphics.gd#48] [Q&A] [answered]
Hi! 

For example we have `GetParent()` returns `Node.Instance` or `GetChildren()` which returns `[]Node.Instance`

But actually our parent is `CharacterBody2D` and we want to get him and use `SetVelocity` method

So how can we cast it?

**Comment by Splizard** [comment, marked as answer]
Hey! Yes you can do this with `Object.As`.
```go
if body, ok := Object.As[gd.CharacterBody2D](node); ok {
    body.SetVelocity(vec)
}
```
